content/_components/microsoft-dynamics-business-central/index.md

```diff
@@ -12,37 +12,88 @@ ComponentVersion: 1.1.1
-|------------------------|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Name                   | true      | your Auth Client's name                                                                                                                                      |
-| Client ID              | true      | your `Application (client) ID`                                                                                                                               |
-| Client Secret          | true      | your `Value` from `Client secrets`                                                                                                                           |
+For creating Auth Client you should specify following fields:
+
+| Field name | Mandatory | Description |
+|------------------------|-----------|-----------------------------------------------------------------|
+| Name                   | true      | your Auth Client's name |
+| Client ID              | true      | your `Application (client) ID` |
+| Client Secret          | true      | your `Value` from `Client secrets` |
 | Authorization Endpoint | true      | set: `https://login.microsoftonline.com/{tenantId}/oauth2/authorize?resource=https://api.businesscentral.dynamics.com`, where `tenantId` is user domain name |
 | Token Endpoint         | true      | set: `https://login.windows.net/{tenantId}/oauth2/token?resource=https://api.businesscentral.dynamics.com`, where `tenantId` is user domain name             |
 
+{% include img.html max-width="100%" url="img/auth-client.png" title="Auth client" %}
+
 - fill field ``Name Your Credential``
 - fill field ``Environment Name`` - required, field indicates what environment to be used. How to get a list of business central environments see [here](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/webservices/api-get-environments). Example: `Production`.
 Base URL for this component is `https://api.businesscentral.dynamics.com/v2.0/${environmentName}/api/v2.0`
 - click on ``Authenticate`` button - the process would take you to Microsoft Dynamics to log-in and give permissions to the platform to access your service.
 - click on ``Verify`` button for verifying your credentials
 - click on ``Save`` button for saving your credentials
 
+### How to get Microsoft Dynamics Business Central instance URL
+
+To access the Microsoft Dynamics Business Central Instance URL, follow these steps:
+
+1. Register on Microsoft Dynamics Business Central:
+   Begin by registering on the [Microsoft Dynamics Business Central platform](https://dynamics.microsoft.com/en-us/business-central/signin/?ru=https%3A%2F%2Fbusinesscentral.dynamics.com%2F%3FnoSignUpCheck%3D1).
```

Comment:
Start by registering on the [Microsoft Dynamics Business Central platform](https://dynamics.microsoft.com/en-us/business-central/signin/?ru=https%3A%2F%2Fbusinesscentral.dynamics.com%2F%3FnoSignUpCheck%3D1).